### PR TITLE
Added contribution guideline

### PR DESCRIPTION
**Pull request by Phil23**

### CODE REVIEW

File: source/contributing/contributing-code/index.md [hunk 1]
```diff
@@ -5,24 +5,20 @@ github_link: contributing/contributing-code/index.md
...
 
 ## Configure Git
 
 Set up your user information with your real name and a working email address:
 
-```bash
-git config --global user.name "Your Name"
-git config --global user.email you@example.com
-```
+```git config --global user.name "Your Name"```
```
Comment (mitelg) [inline source/contributing/contributing-code/index.md]: why have you changed this?

File: source/contributing/contribution-guideline/index.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+<div class="toc-list"></div>
+
+## Introduction
+First of all, thank you! You have decided to contribute code to our software and become a member of the large shopware community. We appreciate your hard work and want to handle it with the most possible respect.
+
+To ensure the quality of our code and our products we have created a small guideline we all should endorse to. It helps you and us to collaborate with our software. Following these guidelines will help us to integrate your changes in our daily workflow. 
+
+## Requirements for a successful pull request
+To avoid that your pull request gets rejected, you should always check that you provided all necessary information, so that we can integrate your changes in our internal workflow very easily. Here is a check-list with some requirements you should always consider when committing new changes.
```
Comment (mitelg) [inline source/contributing/contribution-guideline/index.md]: maybe we should mention here, that they first should look up the existing PRs, so we don't get duplicates :smile:

File: source/contributing/contributing-code/index.md [hunk 3]
```diff
@@ -35,37 +31,35 @@ This will create a "copy" of the entire Shopware repository into your personal u
...
-```bash
-git remote -v
-origin    git@github.com:bcremer/shopware.git (fetch)
-origin    git@github.com:bcremer/shopware.git (push)
-upstream    https://github.com/shopware/shopware.git (fetch)
-upstream    https://github.com/shopware/shopware.git (push)
-```
+```git remote -v```
+
+```origin    git@github.com:bcremer/shopware.git (fetch)```
```
Comment (MarcelSchmaeing) [inline source/contributing/contributing-code/index.md]: bcremer sollten wir mit USERNAME tauschen oder?

File: source/contributing/contribution-guideline/index.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+        <td>
+            Your pull request is incomplete. It is either missing some of the necessary information, or your code implementation is not sufficient to fix the issue. Mostly there will be a comment by our developers which gives you further information of what is missing.
+        </td>
+    </tr>
+    <tr>
+        <td>
+            <img src="/contributing/img/label-tests.jpg" />
+        </td>
+        <td>
+            When you want to commit a new feature or bigger change it is highly necessary that you provide the corresponding tests for it. We only accept new features which are completely tested.
```
Comment (MarcelSchmaeing) [inline source/contributing/contribution-guideline/index.md]: We only accept new features or bug fixes which are completely tested.

File: source/contributing/contributing-code/index.md (hunk 1 above)
Comment (Phil23) [inline source/contributing/contributing-code/index.md]: Layout was a little bit broken. He made an unnecessary line break.

File: source/contributing/contribution-guideline/index.md (hunk 2 above)
Comment (Phil23) [inline source/contributing/contribution-guideline/index.md]: Good Idea! I will add this!

File: source/contributing/contributing-code/index.md (hunk 3 above)
Comment (Phil23) [inline source/contributing/contributing-code/index.md]: Done

File: source/contributing/contribution-guideline/index.md (hunk 4 above)
Comment (Phil23) [inline source/contributing/contribution-guideline/index.md]: Done

File: source/contributing/contribution-guideline/index.md (hunk 2 above)
Comment (Phil23) [inline source/contributing/contribution-guideline/index.md]: Done

**Pull request merged**